purchase_order_menu_filters/views/purchase_order_views.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="purchase.purchase_rfq" model="ir.actions.act_window">
+        <field name="context">{
+            'quotation_only': True,
```

Comment:
Locally tested, :+1: 

But we'll try to determine if this line with this context value `quotation_only` is really needed and why was used in v15. Could you check its usage for v15 and try to find v17 equivalent, if it really exists? If we don't find anything related to this in v17, we should remove it